### DISCUSSION TITLE
something wrong with application webpage

### DISCUSSION THREAD

**Discussion by peterhuang621** [community/community#76384] [GitHub Education]
### Hi everyone,

I'm trying to apply for a student benefit. But at the website which requires uploading verified docs, I find the button for selecting "Proof Type" is missing. And I tried multi devices and browers, all have the same issue. Meanwhile, there is some strange code on web, like the html goes wrong??
I get a right one in a blog:
![image](https://github.com/community/community/assets/30773979/e466e61b-e36a-4e93-ae92-5f4cc57c5eae)
This is what I get as follow:
<img width="877" alt="截屏2023-11-21 12 31 05" src="https://github.com/community/community/assets/30773979/d7e70087-3f1a-4822-9325-3156f7fc36e2">

Could you please fix it? Thank you.

**Comment by YourSonnet18** [comment]
Same problem

**Reply by YourSonnet18** [reply]
and I tried with edge, fiirefox, and on phone, and the problem still exists.

**Reply by peterhuang621** [reply]
Same for me. I tried my phone, ipad but also failed.

**Reply by YourSonnet18** [reply]
No strange codes now, but still missing the proof type option.

**Comment by seliminsan** [comment]
I have same problem :/

![bug](https://github.com/community/community/assets/117529414/f5ad4819-cf1c-4d23-b37d-bdcc592babda)
